features/min_not_in_target_systems.feature

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2017 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Systems in "Configuration => Systems => Target Systems" page
+
+  Scenario: Check absence of the minion in the list
```

Comment:
Why the minion is absent and not present? 

I am just thinking if someday it is present, or if we switch the feature order, can work? why we expect that the minion is absent